apps/site/src/components/Sticker/BaseSticker.tsx

```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+import { motion } from "framer-motion";
+import styles from "./BaseSticker.module.scss";
+import { MutableRefObject, useRef } from "react";
+
+interface StickerProps {
+	imageSrc: string;
+	alt: string;
+	height?: number;
+	width?: number;
+	draggable: boolean;
```

Comment:
draggable and dragConstraints have defaults set. Consider making these props optional.